### PR TITLE
Add Requirements

### PR DESCRIPTION
**Pull request by changeworld**
I understand "NOTE: The documentation is in progress.".
Even in that situation, the current README is not easy to understand for many people.
So I added Requirements.

### CODE REVIEW

**Comment by tricknotes** [comment]
You are our hero!
Thank you so much ✨

File: README.md [hunk 1]
```diff
@@ -4,3 +4,11 @@
...
 
 NOTE: The documentation is in progress.
+
+## Requirements
+
+If you use macOS and have not used groonga:
+
+    $ brew install mecab
+    $ brew install mecab-ipadic
+    $ brew install groonga --with-mecab
```
Comment (cafedomancer) [inline README.md:14]: The dependencies for groonga with the `mecab` option are automatically resolved. So you don't have to run `brew install mecab` and `brew install mecab-ipadic` in advance, right? 😻

screen shot 2016-12-20 at 16 58 29

Comment (tricknotes) [reply README.md:14]: I think you are completely right 👍

Comment (changeworld) [reply README.md:14]: Oops!
I described the steps.
I should only write the steps necessary for the document.
I'm sorry, I will send PR.

**Pull request merged**